wagtail/contrib/redirects/templates/wagtailredirects/choose_import_file.html

```diff
@@ -0,0 +1,43 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n %}
+{% block titletag %}{% trans "Redirects" %}{% endblock %}
+
+{% block extra_js %}
+    {{ block.super }}
+    <script>
+        window.headerSearch = {
+            url: "{% url 'wagtailredirects:index' %}",
+            termInput: "#id_q",
+            targetOutput: "#redirects-results"
+        }
+    </script>
+{% endblock %}
+
+{% block content %}
+    {% trans "Import redirects" as import_str %}
+    {% include "wagtailadmin/shared/header.html" with title=import_str icon="redirect" %}
```

Comment:
I think each page in this imports redirect flow should have the title AND the subtitle to help guide the user through.

e.g.
```
    {% trans "Import redirects" as header_title %}
    {% trans "Choose a file" as header_subtitle %}
    {% include "wagtailadmin/shared/header.html" with title=header_title subtitle=header_subtitle icon="redirect" %}
```